### DISCUSSION TITLE
How can we use pyannote for "streaming" speaker change detection ?

### DISCUSSION THREAD

**Discussion by moadel2002** [pyannote/pyannote-audio#1664] [Q&A]
I am wondering if we can use pyannote for streaming speaker change detection  ?

**Comment by hbredin** [comment]
Not yet but we are working on it! Stay tuned!

👀 @Bilal-Rahou

**Reply by mkl-o** [reply]
Is there any update? I'm also interest to get continuous signal from the stream.
